src/lib/autocomplete/autocomplete.ts

```diff
@@ -98,6 +112,18 @@ export class MatAutocomplete extends _MatAutocompleteMixinBase implements AfterC
   /** Function that maps an option's control value to its display value in the trigger. */
   @Input() displayWith: ((value: any) => string) | null = null;
 
+  /**
+   * Whether the first option should be highlighted when the autocomplete panel is opened.
+   * Can be configured globally through the `MAT_AUTOCOMPLETE_DEFAULT_OPTIONS` token.
+   */
+  @Input()
+  get highlightFirstOption(): boolean { return this._highlightFirstOption; }
```

Comment:
Chatted w/ Miles and Joey, we came to `autoActiveOption` as the proposal for this. I like using "active" since it aligns with the term we use in list-key-manager and in datepicker. Thoughts?